locales/de.json

```diff
@@ -0,0 +1,96 @@
+{
+	"File": "Datei",
+	"Quit": "Schliessen",
+	"Edit": "Bearbeiten",
+	"Undo": "Lösen",
+	"Redo": "Noch einmal",
+	"Cut": "Schneiden",
+	"Copy": "Kopieren",
+	"Paste": "Einfügen",
+	"Delete": "Löschen",
+	"Select all": "Alles markieren //?",
+	"View": "Wiedergabe",
+	"Reload": "Neu laden",
+	"Force Reload": "Neu laden // erzwingen?",
+	"Toggle Developer Tools": "Entwicklerwerkzeuge umschalten",
+	"Actual size": "Wahre Größe",
+	"Zoom in": "Hineinzoomen",
+	"Zoom out": "Rauszoomen",
+	"Toggle fullscreen": "Vollbild umschalten",
+	"Window": "Fenster",
+	"Minimize": "Minimieren",
+	"Close": "Schliessen",
+	"FreeTube": "FreeTube",
+	"Subscriptions": "Abonnements",
+	"Featured": "Vorgestellt",
+	"Most Popular": "Beliebteste ?",
```

Comment:
Why the question mark?

Reply:
Not sure about the translation, forgot the //.